src/EditorFeatures/Core/SymbolMonikers/SymbolMoniker.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Editor.SymbolMonikers
+{
+    internal class SymbolMoniker
```

Comment:
Want to call this an LsifSymbolMoniker just to make it clear this is really not intended for broader use? Not against a rename down the road if it _does_ need to be used for more specific things, but until then this isn't really intended for broad applicability.

Reply:
sure.  What does LSIF stand for?